src/stylesheets/elements/_headings.scss

```diff
@@ -197,3 +197,15 @@ article + aside > h2 {
   margin-top: 0;
   margin-bottom: 0;
 }
+
+/*--- Card styles ---*/
+
+.card,
+[class*="card--"] {
+  > h1, > h2, > h3, > h4, > h5, > h6 {
```

Comment:
Let's remove applying to direct descendent here. We would want all of the headings within a card to look the same otherwise we will have smaller headings to begin with then larger ones.